src/app.js

```diff
@@ -1,3 +1,28 @@
-const app = async () => '#BuildforSDG';
+const express = require('express');
+const bodyParser = require('body-parser');
```

Comment:
# Critical
`body-parser` is already part of `express`. No need to install it separately. Check [here](https://stackoverflow.com/questions/47232187/express-json-vs-bodyparser-json/47232318#47232318) for more context